core-test/src/test/scala/org/bitcoins/core/protocol/script/ConditionalScriptSignatureSpec.scala

```diff
@@ -0,0 +1,28 @@
+package org.bitcoins.core.protocol.script
+
+import org.bitcoins.testkit.core.gen.{NumberGenerator, ScriptGenerators}
+import org.scalacheck.{Prop, Properties}
+
+class ConditionalScriptSignatureSpec
```

Comment:
So let's not use the old scalacheck stuff, and start using scalatest for the property based testing. I would like to migrate all old test cases at some point, but let's not add more burden. Basically just use the `BitcoinSUnitTest` trait here and the normal scalatest framework. 

One thing to note though, since these test cases are new make sure you 

```scala
  implicit override val generatorDrivenConfig: PropertyCheckConfiguration = {
    generatorDriveConfigNewCode
  }
```

which will cause 100 iterations on the property based tests, rather than the scalatest default of 10.